cogs/owner.py

```diff
@@ -800,28 +805,49 @@ def _wait_for_answer(self, author):
             print("The set owner request has been ignored.")
             self.setowner_lock = False
 
+    async def _check_remote_commitlog(self, url, chash):
```

Comment:
You can do this differently.

Invoking `git remote update` will update the remote refs (pointed by the red repo) in which you can then use something like `git status` on it. Bypassing the github API completely. See [here](http://stackoverflow.com/a/3278427/1381108) for more info.